business/core/home/model.go

```diff
@@ -0,0 +1,55 @@
+package home
+
+import (
+	"time"
+
+	"github.com/google/uuid"
+)
+
+// Home represents an individual home.
+type Home struct {
+	ID          uuid.UUID
+	Address     Address
+	Type        string
+	UserID      uuid.UUID
+	DateCreated time.Time
+	DateUpdated time.Time
+}
+
+// Address represents an individual address.
+type Address struct {
```

Comment:
Put the Address type before Home since Home uses it. I like to think of the types as actors and when we can we should present them in the order of appearance.

Reply:
I agree.